fastlane/Fastfile

```diff
@@ -89,7 +90,7 @@ platform :android do
       upload_assets: []
     )
 
-    tag_release_with_latest_if_needed(release_version: release_version)
+    tag_release_with_latest_if_needed(options)
```

Comment:
just curious, why this update? we're not passing anything besides release_version right?

Reply:
Yeah, this change didn't do anything. Should I revert it?

Reply:
i think i prefer the more specific naming. but up to you, we do use options elsewhere in the file 🤷